test/e2e/flask/btc/btc-account-overview.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('BTC Account - Overview', function (this: Suite) {
         assert.equal(await buySellButton.isEnabled(), false);
 
         const portfolioButton = await driver.waitForSelector(
-          '[data-testid="coin-overview-portfolio"]',
+          '[data-testid="coin-overview-receive"]',
```

Comment:
this spec is called `has portfolio button enabled for BTC accounts` however, here we are checking if the receive button is enabled (not the portfolio). 
The portfolio link doesn't appear anymore. Is this expected? If so, the title for the test should be changed, otherwise we should open a bug, indicating that the portfolio link doesn't appear anymore

Reply:
Good catch, PR to fix this:
https://github.com/MetaMask/metamask-extension/pull/28184